Gemfile

```diff
@@ -75,6 +75,7 @@ group :production do
 end
 
 group :test do
+  gem 'simplecov', '~> 0.11.0', require: false
```

Comment:
Is there a reason you're using `~>` and specifying a version? We generally don't lock to a version unless it's necessary.